streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -342,7 +342,7 @@ public void flush(final InternalProcessorContext context) {
     /**
      * @throws IOException if any error happens when closing the state stores
      */
-    public void close(Map<TopicPartition, Long> ackedOffsets) throws IOException {
+    public void close(Map<TopicPartition, Long> ackedOffsets, final boolean writeCheckpoint) throws IOException {
```

Comment:
@guozhangwang i did that already. I think you must have been looking at an outdated diff.